backend/open_webui/models/knowledge.py

```
@@ -89,127 +114,615 @@ class KnowledgeResponse(KnowledgeModel):
+                                if user
+                                else None
+                            ),
+                            collection=self._to_knowledge_model(
+                                knowledge, db=db
+                            ).model_dump(),
+                        )
+                    )
+
+                return KnowledgeFileListResponse(items=items, total=total)
+
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)